src/app/src/test/kotlin/org/vaccineimpact/orderlyweb/tests/database_tests/AuthorizationRepositoryTests.kt

```diff
@@ -225,4 +225,38 @@ class OrderlyWebAuthorizationRepositoryTests : CleanDatabaseTests()
                 .hasMessageContaining("Unknown user : 'user@email.com'")
     }
 
+    @Test
+    fun `getReportReaders gets all users and scopes which can read report`()
+    {
+        JooqContext().use {
+            insertReport("report1", "version1")
+            insertReport("report2", "version2")
+
+            insertUser("global.reader@email.com", "global.reader.name")
+            giveUserGroupPermission("global.reader@email.com", "reports.read", Scope.Global())
+
+            insertUser("scoped.reader@email.com", "scoped.reader.name")
+            giveUserGroupPermission("scoped.reader@email.com", "reports.read", Scope.Specific("report", "report1"))
+
+            insertUser("another.scoped.reader@email.com", "scoped.reader.name")
+            giveUserGroupPermission("another.scoped.reader@email.com", "reports.read", Scope.Specific("report", "report2"))
```

Comment:
Are we ignoring the possibility of version scopes for now?

Reply:
I don't think it's in the scope of the current ticket, which is to reproduce the existing functionality with the existing UI.